src/orquestra/sdk/_base/cli/_dorq/_entry.py

```diff
@@ -405,20 +407,28 @@ def status():
     help="User Token to given server. To generate token, use this command without -t"
     "option first",
 )
-@cloup.option(
-    "--ce",
-    is_flag=True,
-    default=False,
-    help="Log in to Compute Engine. If not passed, will log in to Quantum Engine",
+@cloup.option_group(
+    "Remote Environment",
+    cloup.option(
+        "--qe", is_flag=True, default=False, help="Log in to Quantum Engine. (Default)"
+    ),
+    cloup.option("--ce", is_flag=True, default=False, help="Log in to Compute Engine."),
+    constraint=cloup.constraints.mutually_exclusive,
 )
-def login(config: str, server: str, token: t.Optional[str], ce: bool):
+def login(config: str, server: str, token: t.Optional[str], ce: bool, qe: bool):
     """
     Login in to remote cluster
     """
     from ._login._login import Action
 
+    runtime_name: RemoteRuntime
+    if ce:
+        runtime_name = RuntimeName.CE_REMOTE
+    else:
+        runtime_name = RuntimeName.QE_REMOTE
```

Comment:
**nitpick**: so far we wanted to keep the `_entry` module as slim as possible, in particular devoid of any logic that would need to be tested. This ce/qe seems like a good candidate for an argument resolver. WDYT about moving it there?

Reply:
Seems like a lot of boilerplate, I moved it to `click`'s code via the `flag_value` kwarg in aa7d501. Does that seem better?

What we lose is the `cloup` mutually exclusive support. You can only have a single value for `runtime_name` but `orq login -s ... --ce --qe` would use `--qe` as it was the last option specified.

Reply:
bump

Reply:
> What we lose is the cloup mutually exclusive support.

Yeah that's a good point. Yikes. Pre- 3facd7afdbfaf9d8878c5489443f2b4b43dbc240 seems like a better solution after all.